src/mudlet-lua/lua/generic-mapper/versions.lua

```diff
@@ -5,5 +5,5 @@ return {{[1] = [[2.0]],[2] = [[2.0.1]],[3] = [[2.0.2]],[4] = [[2.0.3]],[5] = [[2
     [13] = [[2.0.12]], [14] = [[2.0.13]], [15] = [[2.0.14]], [16] = [[2.0.15]], [17] = [[2.0.16]], [18] = [[2.0.17]],
     [19] = [[2.0.18]], [20] = [[2.0.19]], [21] = [[2.0.20]], [22] = [[2.0.21]], [23] = [[2.0.22]], [24] = [[2.0.23]],
     [25] = [[2.0.24]], [26] = [[2.0.25]], [27] = [[2.0.26]], [28] = [[2.0.27]], [29] = [[2.1.0]], [30] = [[2.1.1]],
-    [31] = [[2.1.2]], [32] = [[2.1.3]], [33] = [[2.1.4]], [34] = [[2.1.5]], [35] = [[2.1.6]]
+    [31] = [[2.1.2]], [32] = [[2.1.3]], [33] = [[2.1.4]], [34] = [[2.1.5]], [35] = [[2.1.6]], [36] = [[2.1.7]],
```

Comment:
Um, doesn't the number here have to match the version number declared in the file i.e. `2.2` (or possibly `2.2.0`)?